js/models/languagesMd.js

```diff
@@ -253,9 +256,9 @@ module.exports = Backbone.Model.extend({
         ReadOnly: "(This Field is Read Only)",
         HandleResolver: "Handle Resolver",
         moderatorSettings: {
-          ProvideResolution: "Provide dispute resolution",
+          DisputeResolution: "Dispute Resolution",
           ServiceFee: "Service fee",
-          ServiceFeeNote: "Percentage of transaction price (max 25, step 0.01, min 0)"
+          ServiceFeeNote: "Min: 0%, Max: 25%"
```

Comment:
I think we should have something here that explains why they can't put in 0.001%. Maybe a placeholder?